java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -247,6 +258,40 @@ private boolean isConstraintFromObjectDomain(@Nullable Constraint constraint) {
       return constraint instanceof ObjectConstraint;
     }
 
+    private List<Location> flowForNullableMethodParameters(ExplodedGraph.Node node) {
+      if (node.parent() != null || !domains.contains(ObjectConstraint.class)) {
+        return ImmutableList.of();
+      }
+      ImmutableList.Builder<JavaFileScannerContext.Location> flowBuilder = ImmutableList.builder();
```

Comment:
Agreed, but I added it by mistake (see line [261](https://github.com/SonarSource/sonar-java/pull/1448/files#diff-ce6001fad62a230360636a1563f880ecR261)). I'm going to remove it. Don't want to add changes everywhere.